english/critter/critter.txt

```diff
@@ -0,0 +1,5 @@
+A critter approaches, wiggling its ears and sniffing, seeming to know that you have recently eaten a sandwich. Do you 
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.